data/strings/strings.txt

```diff
@@ -28164,3 +28164,46 @@
     tr = Google Play'deki yeni kısıtlamalar nedeniyle OpenStreetMap yetkilendirmesi geçici olarak kullanılamıyor 😾\nEkibimiz bu sorunu çözmek için çalışıyor. Lütfen sabırlı olun 🙏
     uk = Авторизація на OpenStreetMap тимчасово недоступна через нові обмеження в Google Play 😾\nНаша команда працює над вирішенням цієї проблеми. Будь ласка, наберіться терпіння 🙏
     zh-Hans = 由于 Google Play 的新限制，OpenStreetMap 授权暂时不可用 😾n我们的团队正在努力解决这个问题。请耐心等待 🙏
+
+  [placepage_less_button]
+    tags = android
+    en = Less
+    af = Minder
+    ar = أقل
+    az = Daha az
+    be = কম
+    bg = По-малко
+    ca = Menys
+    cs = Méně
+    da = Mindre
+    de = Wenig
+    el = Λιγότερο
+    es = Menos
+    et = Less
+    eu = Less
+    fa = کمتر
+    fi = Vähemmän
+    fr = Moins
+    he = פָּחוֹת
```

Comment:
This includes nikkud which is very rarely used in apps and daily life, and can be off-putting. Replace with:

he = פחות

I'm curious, what precisely are you using to translate? Maybe there's a tweak we can add to avoid nikkud in the future. (Or I'll just keep correcting it when it shows up, that works too...)

Reply:
I opened an issue upstream https://github.com/soimort/translate-shell/issues/509, assuming you used translate.py. Regardless, can you fix the translation for this PR in the meantime?

Reply:
Which script are you using? `python/tools/translate.py` provides different results for me (note be and zh). I've also manually corrected some translations:

```
    en = Less
    af = Minder
    ar = أقل
    az = Daha az
    be = Менш
    bg = По-малко
    ca = Menys
    cs = Méně
    da = Mindre
    de = Weniger
    el = Λιγότερο
    es = Menos
    et = Vähem
    eu = Gutxiago
    fa = کمتر
    fi = Vähemmän
    fr = Moins
    he = פחות
    hi = कम
    hu = Kevesebb
    id = Kurang
    it = Meno
    ja = より少ない
    ko = 더 적은
    lt = Mažiau
    mr = कमी
    nb = Mindre
    nl = Minder
    pl = Mniej
    pt = Menos
    ro = Mai puțin
    ru = Меньше
    sk = Menej
    sv = Mindre
    sw = Chini
    th = น้อย
    tr = Daha az
    uk = Менше
    vi = Ít hơn
    zh-Hans = 更少
    zh-Hant = 較少的
```

Reply:
Done